woocommerce/woocommerce-android
# Feature/8861 new signup flow

Closes: #8861 


### Description

Splits sign up feature into 2 steps: 
- Enter email
- Enter password

Design details can be found here: aLjmEW0tq6nLwaA1NYXsq1-fi-241_21811

![Screenshot 2023-04-24 at 15 21 32](https://user-images.githubusercontent.com/2663464/234008920-b34c6fea-2548-4a84-9da7-0df5f93f9520.png)

The reason for this is we want to check if the entered email already belongs to an existing WordPress account. If that is the case, the user will be directly taken to the log in screen. 

### Testing instructions

1. Click on “Get Started” from the log in prologue
2. Enter an invalid email and click continue -> Check invalid email error
3. Enter an existing email and check that you are taken to Log in with email screen with the prefilled email
4. Navigate back and enter an email that doesn't belong to WordPress account
5. Enter a numeric password only and check password invalid error
6. Enter a password shorter than 7 characters and check password invalid error
7. Enter a valid password and check that store creation flow is triggered right away. 
8. Navigate back from store creation flow and check you land on site picker.
